leetcode/array.md

```diff
@@ -0,0 +1,22 @@
+# Array
```

Comment:
Your pull request: LEETCODE-1022 Added max-consecutive-ones


Your commit: LEETCODE-1022 Added max-consecutive-ones

Your commit: LEETCODE-1022 Refactored max-consecutive-ones
Action of your comment must be from ['Added', 'Deleted']